modules/database/sqlite/db.copy

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * This file is auto-generated and copied from the performance-lab plugin.
+ *
+ * Please don't edit this file directly.
+ *
+ * @package performance-lab
+ */
+
+// Bail early if the SQLite implementation was not located in the performance-lab plugin.
+if ( ! file_exists( '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php' ) ) {
+	return;
+}
+
+// Define SQLite constant.
+if ( ! defined( 'DATABASE_TYPE' ) ) {
+	define( 'DATABASE_TYPE', 'sqlite' );
+}
+
+// Require the implementation from the performance-lab plugin.
+require_once '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php';
```

Comment:
The help for `wp plugin delete` is clear about deleting "the plugin's files without deactivating or uninstalling it" first.
In the case of Query Monitor, it leaves the drop-in (db.php, the same one used here) in place.  I think this has to be treated as operator error. wp-cli can be a chainsaw.

Reply:
I think if we change `require_once` with `include_once`, then unlucky self foot shooters won't get fatal error because of this require.

```suggestion
include_once '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php';
```

Reply:
> If I delete the performance plugin using the `wp plugin delete {performance-plugin}` command, then my site will go down immediately due to a fatal error and I won't be able to recover it.

No, the site won't go down. 
A few lines above this, we check if the file exists and if it doesn't then we bail early - at which point the site will fallback to its previously used MySQL database with no errors.

Reply:
Oh.. my bad... you are right... I missed the line that checked whether the file exists and returns early. Anyway the `file_exists` function is IO bound and always checking filesystem for every request makes the site slightly slower. I don't say that we need to remove it, but if we can find a way how to avoid using this function, then it would be better.

Reply:
>  the `file_exists` function is IO bound and always checking filesystem for every request makes the site slightly slower.

I agree 100%! ❤️ 
Ideally we wouldn't need that call, and when SQLite gets merged in Core, we won't. This is a temporary measure to ensure nothing breaks while the implementation is still in a plugin. By necessity, this is not as optimized as it will be when merged in Core.